internal/kafka/internal/services/cluster_placement_strategy.go

```diff
@@ -58,11 +59,15 @@ func (f *FirstSchedulableWithinLimit) FindCluster(kafka *dbapi.KafkaRequest) (*a
 	criteria := FindClusterCriteria{
 		Provider:              kafka.CloudProvider,
 		Region:                kafka.Region,
-		MultiAZ:               kafka.MultiAZ,
```

Comment:
This might not have the effect we want as well, since the default value when a boolean isn't set is `false`.